### ISSUE TITLE
Support Google-Java-Formats own import order out of the box

### ISSUE THREAD

**Issue by bjrke** [diffplug/spotless#1769]
As mentioned in #1680 spotless v6.18.0 unintentionally switched to gjf's own import order handling which was introduced as default also into the corresponding intellij plugin.
With spotless v6.19.0 the import order was changed back to the previous behavior of spotless prior v6.18.
I would like to see a simple config API change to the `googleJavaFormat()` definition which disables spotless own import order and uses the order from gjf.

**Comment by nedtwigg** [comment]
Implementing this is super duper easy, just do the inverse of this: https://github.com/diffplug/spotless/pull/1680/commits/0183cc20385361775fac3eacc0b11f8a443f4c0f

The only tricky part is the API, maybe: `googleJavaFormat().reorderImports(true)`, and leave `false` as the default value.

**Comment by nedtwigg** [comment]
Released in `plugin-gradle 6.21.0` and `plugin-maven 2.38.0`.

**Comment by bjrke** [comment]
I tried this out with the recently release version 6.21 of the gradle plugin.
It looks like the new setting `reorderImports(true)` doesn't have any effect on our setup. While others like `aosp()` or using an older version is working.

I get the following change in the import order when updating from 6.18.0 (which accidentally enabled gjfs own import order) to 6.21.0:
```
          -
          -import·org.slf4j.Logger;
          -import·org.slf4j.LoggerFactory;
          -
           import·java.nio.file.Path;
           import·java.nio.file.Paths;
           import·java.util.Locale;
           import·java.util.Optional;
           import·java.util.stream.Stream;
          +import·org.slf4j.Logger;
          +import·org.slf4j.LoggerFactory;
```
